scripts/php-syntax

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
-find . -type f -name '*.php' -not -path '*/vendor/*' -print0 | xargs -0 -n1 php -l
+# The -P10 option specifies the number of parallel processes (In constrainted CPUs will take approx time for 1 available cpu)
+find . -type f -name '*.php' -not -path '*/vendor/*' -print0 | xargs -0 -n1 -P10 php -l
```

Comment:
@syedahmedhaidershah please redirect the STDOUT to `/dev/null` 

In my testing I found that this would send a lot of information to STDOUT that I would argue is unnecessary. We do not care if the linter did not find issues with the files but only care if they do. 

I tested this in a couple of repos and found that it can save more time as well. 

With STDOUT redirect:
```
find . -type f -name '*.php' -not -path '*/vendor/*' -print0  0.02s user 0.10s system 1% cpu 9.813 total
xargs -0 -n1 -P10 php -l > /dev/null  36.09s user 20.03s system 572% cpu 9.811 total
```
Without:
```
find . -type f -name '*.php' -not -path '*/vendor/*' -print0  0.02s user 0.11s system 0% cpu 13.204 total
xargs -0 -n1 -P10 php -l  36.35s user 21.14s system 435% cpu 13.202 total
```

Reply:
Absolutely ✅

Reply:
Stdout is being directed to `/dev/null` now